mtags/src/main/scala/scala/meta/internal/mtags/MtagsEnrichments.scala

```diff
@@ -98,6 +98,15 @@ trait MtagsEnrichments {
         case _ => false
       }
     }
+    def isWorksheet: Boolean = {
+      filename.endsWith(".worksheet.sc")
+    }
+    def isScalaOrScalaScript: Boolean = {
+      extension match {
+        case "scala" | "sc" => true
```

Comment:
I'm a bit confused in general on whether this feature supports `.sc` or `.worksheet.sc`.

Reply:
Completions work in `*.sc` files even if they are not `*.worksheet.sc`. The "eval on file save and display decorations" feature only works for `*.worksheet.sc`

Reply:
The new `isScalaFilename` method name could make it clearer

* "scala filename": completions/hover/... work
* "scala worksheet": eval with mdoc and display decorations